packages/account/src/Components/poa/status/unverified/unverified.tsx

```diff
@@ -1,21 +1,18 @@
 import React from 'react';
-import { PlatformContext } from '@deriv/shared';
 import { Button, Icon, Text } from '@deriv/components';
 import { localize } from '@deriv/translations';
-import IconMessageContent from 'Components/icon-message-content';
+import IconMessageContent from '../../../icon-message-content';
 
 type TUnverified = {
     onClick: () => void;
 };
 
 export const Unverified = ({ onClick }: TUnverified) => {
-    const { is_appstore } = React.useContext(PlatformContext);
-
     return (
         <IconMessageContent
             message={localize('We could not verify your proof of address')}
             text={localize('Please check your email for details.')}
-            icon={<Icon icon={is_appstore ? 'IcPoaErrorDashboard' : 'IcPoaError'} size={128} />}
+            icon={<Icon icon='IcPoaError' size={128} />}
```

Comment:
Please remove `IcPoaErrorDashboard` icon file as well

Reply:
I didn’t find it. it seems there was no such icon at all :)